src/delivery-options/config/configBus.js

```diff
@@ -213,6 +213,7 @@ export const createConfigBus = (eventCallee = null) => {
           this[item] = configuration[item];
         });
 
+        this.currentCarrier = Object.keys(configuration.config.carrierSettings)[0];
```

Comment:
iets om over na te denken;

Wellicht handig om deze selectie te centraliseren naar een helper functie of carrier Model.
Het is gek dat je een carrier haalt uit de carriersettings.

En stel dat deze carriersettings qua structuur wijzigen, dan zal je het om meerdere plekken moeten wijzigen.

Reply:
Zal het zeker in m'n achterhoofd houden bij de volgende scope creep 😄  Zou dit ook graag anders zien